File: client/src/app/models/reportingActivite.ts
export class ReportingActivite {


  nomActivite: string;
  groupe: string;
  nbInitee: number;
  nbPayee: number;
  nbValidee: number;
  nbF: number;
  nbM: number;
  cotisations: number;



  constructor() {

    this.nomActivite = "";
    this.groupe = "";
    this.nbInitee = 0;
    this.nbPayee = 0;
    this.nbValidee = 0;
    this.nbF = 0;
    this.nbM = 0;
    this.cotisations = 0;
  }
}

File: app/src/main/java/com/wild/corp/adhesion/models/ReportingActivite.java
package com.wild.corp.adhesion.models;

import lombok.*;

@Getter
@Setter
@ToString
@Builder
@AllArgsConstructor
@RequiredArgsConstructor
public class ReportingActivite {

    private String nomActivite;

    /** Groupe métier affiché comme en-tête et utilisé pour les sous-totaux. */
    private String groupe;

    private Long nbInitee =0L;

    private Long nbPayee =0L;

    private Long nbValidee =0L;

    private Long nbF =0L;

    private Long nbM =0L;

    private Long cotisations =0L;

}
